src/sentry/static/sentry/app/views/acceptProjectTransfer.tsx

```diff
@@ -7,9 +8,21 @@ import NarrowLayout from 'app/components/narrowLayout';
 import SelectField from 'app/views/settings/components/forms/selectField';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
 import {t, tct} from 'app/locale';
+import {Organization, Project} from 'app/types';
 
-class AcceptProjectTransfer extends AsyncView {
-  getEndpoints() {
+type Props = RouteComponentProps<{}, {}>;
+
+type TransferDetails = {
+  organizations: Organization[];
+  project: Project;
+};
+
+type State = {
+  transferDetails: TransferDetails;
```

Comment:
```suggestion
  transferDetails: TransferDetails | null;
```

The AsyncComponent sets it to null in case of error 😉